Makefile

```diff
@@ -4,12 +4,18 @@ COMPILED_TAG_NAME ?= latest
 ENVIRONMENT_TAG_NAME ?= latest
 COMPILE_OPTION ?=
 COMPILE_TARGET ?= fv3gfs-compiled
+BUILD_ARGS ?=
+BUILD_FROM_INTERMEDIATE ?= n
```

Comment:
The name is clear, though a bit long if you type it out on the command-line -- probably good though, I don't have a shorter suggestion that is as clear.

Reply:
If you plan to use this in your personal work, it's probably worth setting default to "y".